### ISSUE TITLE
Error occurred while proxying request localhost:3000 to http://localhost:7000

### ISSUE THREAD

**Issue by papandadj** [opencv/cvat#6118]
### My actions before raising this issue
- [y] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [y] Searched [past issues](/issues)

I think the problem I encountering is different from this one #5605

### Steps to Reproduce (for bugs)

1. Install OPA and pg, It's work.
1. Start python server with `ALLOWED_HOSTS="*" IAM_OPA_BUNDLE="1" python manage.py runserver --noreload --insecure 0.0.0.0:7000`, It's work.
1. Fellowing the step above, I can access API `http://localhost:7000/api/server/health/?format=json&org=`
<img width="1670" alt="image" src="https://user-images.githubusercontent.com/25424898/237026371-0a25864a-0cb3-4962-982f-a8c7a8f0fcab.png">
1. Start `yarn start:cvat-ui` and access `http://localhost:3000/`
<img width="834" alt="image" src="https://user-images.githubusercontent.com/25424898/237026702-585b1873-30d5-4a30-9af1-83b1ba89a64d.png">

### Expected Behaviour
I have started the server and it can be accessed, but the UI still mistakenly. What is possible reason for the error.

### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05):
- Are you using Docker Swarm or Kubernetes?
- Operating System and version (e.g. Linux, Windows, MacOS):
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

**Comment by papandadj** [comment]
I change the listening IP with 127.0.0.1 for launching Python, but still got the same error. I am currently using m1 chip. later, i will switch to Ubuntu and try again.

**Comment by papandadj** [comment]
This is very strange. I downgraded Nodejs from version 18 to version 16 and now it's work locally.
https://stackoverflow.com/questions/62482126/webpack-devserver-proxy-to-https-giving-error-occurred-while-trying-to-proxy-re/75057952#75057952

When I use Nodejs version 18 and access a remote Python backend, It can be accessed successfully. The Python backend start with docker-compose and command is `docker compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.serverless.yml up -d.` I have modified the API_URL="http://remoteIP:8080" in cvat-ui/package.json using Nodejs 18 start. It's work.
